service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C29RecoveryOfAChildDocumentParameterGeneratorTest.java

```diff
@@ -102,10 +106,13 @@ void shouldGenerateDocumentWithEntryMessage() {
         List<Element<Child>> selectedChildren = wrapElements(child);
 
         when(childrenService.getSelectedChildren(caseData)).thenReturn(selectedChildren);
+        when(manageOrderDocumentService.getChildGrammar(selectedChildren.size())).thenReturn("child");
 
         DocmosisParameters generatedParameters = underTest.generate(caseData);
         DocmosisParameters expectedParameters = expectedCommonParameters()
-            .orderDetails(getStandardMessage(selectedChildren.size(), PlacedUnderOrder.CARE_ORDER)
+            .orderDetails(getStandardMessage(
```

Comment:
I'd extract the standard message into a variable. It would make reading easier, IMO.